src/makeRobotAccountant.js

```diff
@@ -6,7 +6,25 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  function getSum(x) {
+    return (y) => {
+      count++;
```

Comment:
Use arrow functions to improve code readability and compression:
`const getSum = (x) => (y) => {
  count++;
  ...
`